website/notifications/tasks.py

```diff
@@ -46,9 +48,76 @@ def send_users_email(send_type):
                 )
             remove_notifications(email_notification_ids=notification_ids)
 
+    # Send emails to reviews moderators
+    grouped_emails = get_moderators_emails(send_type)
+    for group in grouped_emails:
+        user = OSFUser.load(group['user_id'])
+        provider = AbstractProvider.objects.get(id=group['provider_id'])
```

Comment:
It looks like this line can go in the if block to avoid extra work if the user is disabled.